.travis.yml

```diff
@@ -17,18 +17,25 @@ script:
 # Assuming you have installed the travis-ci CLI tool, after you
 # create the Github repo and add it to Travis, run the
 # following command to finish PyPI deployment setup:
-# $ travis encrypt --add deploy.password
+# If your project is built on travis-ci.org you can use
+# $ travis encrypt your-password-here --add deploy.password
 # Or just use this website:
 # Repository name: AllenCellModeling/aicsimageio
 # http://rkh.github.io/travis-encrypt/public/index.html
+# If your project is built on travis-ci.com you can use
+# $ travis encrypt your-password-here --add deploy.password --com
+#
+# Details here:
+# https://docs.travis-ci.com/user/deployment/pypi/
 deploy:
+  skip_cleanup: true
+  skip_existing: true
   provider: pypi
   distributions: sdist bdist_wheel
   user: aicspypi
   password:
-    secure: Br8fLgBI4xZoM+suSb6yylh/WY6YC4CKIB3/D+4tBRCpVuhs/KrNU1ZZIe6Q9HX6t6D1WNz7VKHc5qnDK8Ld4dN3M7Uxb3fqZxHDkKWiCjGZA5ALtzFeUOLzqFjbNhEjTGbErbyRt93br2cIL+PYHDRvN7CRTb41adb9cA1aRXQCs2qhvaahpaxD+XzqbnZQbzBuDK3TWtX/6kgdV1tLv0HSw2wbXlaC1ylYZJ5opjzZRrOAbBE/WHiXZsLK7x0Tgy2hLU77nIg83DzXhD61rkK+V9r1TUpP/JQeqdBcJgwEPyIi0M2GrsQG3Npcjv8G6jeylFQiW7qQQlRXT1PE+NrOHYRD1YuzRDNMS5OtMfc5sIMSOJntfJbqhKUw4cbGp9VXUaez1bGmrQ8aAb/99SesMES/1de09XjZKdpLm2mcJwaQdfncEYl95X9qdVGDdrDq39vY4UsZltkA8DuqgVkZgIXeZn/FIn9y/YmYn/5TF2cH5Ds10X0LAeFrIyzWhETBJo3s2AayGxSfkJsT73n+YW8Ic1FtLQ5jRzQZ6BApSoAup2/4za4YGQ5f8Rva7OejutFQrmHmfC30T01FQGkeEvcoBV/78XLT/zmoema/UyOd36WpJHdcVTHaXoNBd6vxRMnLfu5u02wyzFUfnwoVTJqHdud9j/0xOc9osUo=
+    secure: wnLqFMs7RBxSw94HR72bMu9wplrPUgASPgP+U1ur5T+MUy8xBjgOn5qSzh+LzyJ4y9ziQ9VMI+JyYdBqLpp+vrYpNuh1vq+Tv4o/hLrLj40WJs1DyqgglZSmTwHnjXgRwrCxg4ctHfrbsH+pFPmwpqWp1fbA9FPiPtUFTdAMf//IfXtsbPZlOru7HSMnn5mHSuRV74Bzp1tUHDr4QeBchLP4aen9KyGOcMrw76UqPXHrg0ESCqh/GKeNcGhsFk/pJbYfUUkdmNirZEVCUF68L7GJHWw43qs7ekXyt2E2qk8a3rz8t8R2wOuOK418u3/KOmM3NBb20r+CijyYc44wtvLNUgS9YLA8Do01DLF19+MExWS6XDWRcwUI5BMAjx8ajBFeqOYB2eiwqYRyb+K2ujoq2380PBDb2bIoZ/DqpyGuMv/Py5P+Dby86m+cXExC/hkDJ2/PT1Pn6RKZWHi1D12fnZ6fQfTdSpKgdY5Gay42SuaphmrCSDAWydqOikUEYusob3oDLkjKmJPAvsHR+tMn7KBQ2sLtpbYQn0L9yc4Jk4mxZnL4oBvZP2FnPLtatQLN6ztBBUq2fr4u7RAmKlpxi9i1wLpnFkh217lV61zS428htCEkBtF/Jvr7pz9myoULPMYdLExg8p+Z+j9XbkuuaQ1rKuV0KJo5Zd1XyMQ=
   on:
-    branch: master
```

Comment:
I think this should probably still be present in the new config

Reply:
So unfortunately we can't. This was one of the reasons the release didn't happen on the other branch. We can either do 'on branch' or 'on tag' but not both. :'(

Reply:
Just updated the PR with a new release scheme. We have a stable branch for a reason :^)